plugins/account_notes/account_notes_evaluators.cpp

```diff
@@ -0,0 +1,68 @@
+#include <golos/plugins/account_notes/account_notes_operations.hpp>
+#include <golos/plugins/account_notes/account_notes_objects.hpp>
+#include <golos/plugins/account_notes/account_notes_evaluators.hpp>
+#include <golos/chain/account_object.hpp>
+
+namespace golos { namespace plugins { namespace account_notes {
+
+using golos::chain::from_string;
+
+bool set_value_evaluator::is_tracked_account(const account_name_type& account) {
+    if (!settings_->tracked_accounts.empty()
+        && std::find(settings_->tracked_accounts.begin(), settings_->tracked_accounts.end(),
+            std::string(account)) == settings_->tracked_accounts.end()) {
+        return false;
+    }
+
+    if (std::find(settings_->untracked_accounts.begin(), settings_->untracked_accounts.end(),
+            std::string(account)) != settings_->untracked_accounts.end()) {
+        return false;
+    }
+
+    return true;
+}
+
+void set_value_evaluator::do_apply(const set_value_operation& o) {
+    try {
+        if (!is_tracked_account(o.account)) {
+            return;
+        };
+
+        db().get_account(o.account);
```

Comment:
You already check account on exist in `get_required_active_authorities` of operation.